Explorer/Assets/DCL/PluginSystem/Global/GenericPopupsPlugin.cs

```diff
@@ -0,0 +1,81 @@
+using Arch.SystemGroups;
+using Cysharp.Threading.Tasks;
+using DCL.AssetsProvision;
+using DCL.Clipboard;
+using DCL.UI;
+using MVC;
+using System;
+using System.Threading;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
+using UnityEngine.Serialization;
+
+namespace DCL.PluginSystem.Global
+{
+    public class GenericPopupsPlugin : IDCLGlobalPlugin<GenericPopupsPlugin.Settings>
+    {
+        private readonly IAssetsProvisioner assetsProvisioner;
+        private readonly IMVCManager mvcManager;
+        private readonly IClipboardManager clipboardManager;
+
+        private PastePopupToastController? pasteToastButtonController;
+        private ChatEntryMenuPopupController chatEntryMenuPopupController;
```

Comment:
I suppose you should dispose it as well and make it nullable, or make it a local variable as you don't use it as a field currently